deploy/.gitignore

```diff
@@ -1 +1,2 @@
 ccm-linode.yaml
+.DS_Store
```

Comment:
Minor nitpick: Leave the newline at the end of files.

Passing along the link Tim(🙏😇) sent me 🙂: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
Added a new line